r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -418,3 +418,24 @@ test_that("mutate and write_dataset", {
       summarize(mean = mean(integer))
   )
 })
+
+test_that("mutate and pmin/pmax", {
+  df <- tibble(
+    city = c("Chillan", "Valdivia", "Osorno"),
+    val1 = c(200, 300, NA),
+    val2 = c(100, NA, NA),
+    val3 = c(0, NA, NA)
+  )
+
+  expect_dplyr_equal(
+    input %>%
+      mutate(
+        max_val_1 = pmax(val1, val2, val3),
+        max_val_2 = pmax(val1, val2, val3, na.rm = T),
+        min_val_1 = pmin(val1, val2, val3),
+        min_val_2 = pmin(val1, val2, val3, na.rm = T)
+      ) %>%
+      collect(),
+    df
+  )
```

Comment:
Could you add some additional code in this test that uses a mix of:
- scalar literal values
- column references
- expressions

in the `pmin()` and `pmax()` calls? For example:

```r
pmax(df$val1, 250, df$val2 * 4, df$val3, 50)
```

Reply:
adding